pkg/dockerregistry/server/metrics/prometheus.go

```diff
@@ -42,43 +42,39 @@ var (
 		},
 		[]string{"code", "method"},
 	)
-	HTTPRequestDurationSeconds = prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
+	HTTPRequestDurationSeconds = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
 			Namespace: namespace,
 			Subsystem: httpSubsystem,
 			Name:      "request_duration_seconds",
 			Help:      "A histogram of latencies for requests to the registry.",
```

Comment:
help text is wrong now

Reply:
a summary is a histogram

Reply:
hm.  prometheus docs treat them as two different things with slightly different behaviors:

https://prometheus.io/docs/practices/histograms/

but ok.

Reply:
Not ok, math treats them differently too. A summary is not a histogram, it is a quantile.